admin/class-suggested-plugins.php

```diff
@@ -149,15 +149,17 @@ protected function create_more_information_link( $url, $name ) {
 	 * @return string The activate suggested plugin message.
 	 */
 	protected function create_activate_suggested_plugin_message( $suggested_plugin, $third_party_plugin ) {
```

Comment:
Same here: the second argument is not used anymore so can be removed.

It can also be removed from the calls to this method.
```suggestion
	protected function create_activate_suggested_plugin_message( $suggested_plugin ) {
```